odfuzz/entities.py

```diff
@@ -108,14 +108,15 @@ def _append_corresponding_queryable(self, query_group):
 
 class DirectBuilder:
     """A class for building and initializing all queryable entities with metadata passed in constructor."""
-    def __init__(self, metadata, restrictions,method):
+    def __init__(self, metadata, restrictions,method, sap_vendor_enabled = False):
```

Comment:
I am thinking more about not adding too specific parameter - like the boolean flag here - but entire configuration dictionary, to build the Config class from (line 117).   

For the purpose of testing this change, I am OK with merging the PR as is, however created issue #107 addressing this concern.